test/e2erp/realrp_test.go

```diff
@@ -14,4 +23,72 @@ var _ = Describe("Resource provider e2e tests [Real]", func() {
 		var err error
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	It("should not be possible for customer to mutate an osa scale set", func() {
+
+		ctx := context.Background()
+		ctx = context.WithValue(ctx, api.ContextKeyClientID, os.Getenv("AZURE_CLIENT_ID"))
+		ctx = context.WithValue(ctx, api.ContextKeyClientSecret, os.Getenv("AZURE_CLIENT_SECRET"))
+		ctx = context.WithValue(ctx, api.ContextKeyTenantID, os.Getenv("AZURE_TENANT_ID"))
+
+		logrus.SetLevel(log.SanitizeLogLevel("Debug"))
+		logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
+		logger := logrus.WithFields(logrus.Fields{"location": c.location, "resourceGroup": c.resourceGroup})
+
+		arg := azureclient.ApplicationResourceGroup(c.resourceGroup, c.resourceGroup, c.location)
```

Comment:
arg is not very intuitive. maybe appRg, appRG or something along those lines is clearer

Reply:
done ...changed to `appRg`